io/io/src/TBufferMerger.cxx

```diff
@@ -21,7 +21,7 @@ namespace ROOT {
 namespace Experimental {
 
 TBufferMerger::TBufferMerger(const char *name, Option_t *option, Int_t compress)
-   : fName(name), fOption(option), fCompress(compress),
+   : fName(name), fOption(option), fCompress(compress), fAutoSave(0),
```

Comment:
I'd suggest to set this to != 0, i.e. to enable buffering by default. 30MB seems to be a common number ;-)

Reply:
Well the 'default' for autoSave is 300MB (i.e 10 times AutoFlush).

Reply:
So, should I set it to 300MB then? What do you think, @Axel-Naumann?

Reply:
In your performance test which value did you use?

Reply:
I used a very large value, 1GB I think. Not something we can use as default.

Reply:
I just wanted to make sure that it would make the compression work go away.